icu4c/source/i18n/unicode/rbtz.h

```diff
@@ -89,17 +89,18 @@ class U_I18N_API RuleBasedTimeZone : public BasicTimeZone {
     virtual bool operator!=(const TimeZone& that) const;
 
     /**
-     * Adds the <code>TimeZoneRule</code> which represents time transitions.
-     * The <code>TimeZoneRule</code> must have start times, that is, the result
-     * of isTransitionRule() must be true. Otherwise, U_ILLEGAL_ARGUMENT_ERROR
+     * Adds the `TimeZoneRule` which represents time transitions.
```

Comment:
optional: I prefer not Markdown-ifying API docs that should be about the same between C++ and Java, because it makes it harder to keep them in sync.

Reply:
I'd like to leave it markdown. The Java and C++ docs are sufficiently different that there's copy/paste isn't going to work anyhow.